src/components/daceListener.ts

```diff
@@ -0,0 +1,57 @@
+import * as Net from 'net';
```

Comment:
I feel like this file and `breakpointHandler.ts` should live in the `debugger` directory, as they aren't really view components, no?

Reply:
You are right, I agree, moved them